### ISSUE TITLE
Add keyboard layout switcher

### ISSUE THREAD

**Issue by Der-Meister-von-codes** [dessalines/thumb-key#94]
Add an action to the numeric mode change key (#) to change the current keyboard.  Up - next, and down - previous, respectively (you can also add a pop-up window on hold, as it works on the spacebar in the default android keyboards).  As for your arguments about the inexpediency of this operation due to the possibility of switching the language from the settings to the previous request to add such functionality - this is terribly inconvenient, because a new window opens, the language selection button is pressed, the language is selected, and only after that you can go to  text input field and type the desired one (this is especially true for weak phones that spend a lot of time on this kind of action, or people who need to switch between several languages ​​bwith lightning speed).  I don’t think that one button with a swipe up / down will greatly overload the system with an abundance of actions.
